base/bin/migrate.mustache

```diff
@@ -0,0 +1,27 @@
+const migrate = require('root/lib/migrate');
+const r = require('root/lib/db');
+
+const direction = process.argv[2];
```

Comment:
That would make yargs a dep of any generated app, whereas right now it's only a dep of the CLI. I'd rather keep the generated package.json as lean as possible.